java/pom.xml

```diff
@@ -659,7 +659,11 @@
         <exclusions>
           <exclusion>
             <groupId>io.netty</groupId>
-            <artifactId>netty</artifactId>
+            <artifactId>netty-handler</artifactId>
```

Comment:
Just wondering what is the reason behind the exclusion change here?

Reply:
`Zookeeper` dependency changed. Previously, Zookeeper 3.4 depends on `io.netty:netty`. Now, Zookeeper 3.6 depends on these two.
- https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper/3.6.2